board/recalbox/fsoverlay/recalbox/share_init/system/.emulationstation/es_systems.cfg

```diff
@@ -67,6 +67,7 @@
 		    <core>gliden64</core>
 		    <core>n64</core>
 		    <core>rice</core>
+    		    <core>glide64mk2</core>
```

Comment:
Add a post install sed for xu4 only that adds the core ? 
It's the first platform that is not a pi, so maybe we should consider handling some files like the recalbox.conf afterall : per platform